asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm

```diff
@@ -195,25 +195,25 @@ <h1>アドバンストサーバーアクセス</h1>
                                                             <p class="tile-subtitle"><span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>のチームをセットアップし、<span class="mc-variable okta-variables.ProductName variable">Okta</span>からユーザーとグループをプッシュします。</p>
                                                         </div>
                                                         <div>
-                                                            <p class="tile-title">サーバーエージェント</p>
+                                                            <p class="tile-title"><a href="sftd.htm">サーバーエージェント</a></p>

```

Comment:
What happened here? Did XTM removed the `<a>` elements or the version we sent did not have it?

Reply:
We had missing links in the tiles at the time that we sent the project out (the issue was in source EN, it's been fixed).